### DISCUSSION TITLE
How to obtain coordinates of points on a cross-section?

### DISCUSSION THREAD

**Discussion by MindyL1** [neperfepx/neper#851] [Q&A] [answered]
As shown in the image, I am able to visualize a rough cross-section inside a cube by filtering. I would like to export the shape of this cross-section (in point form). Is there any way to achieve this?
![img7](https://github.com/neperfepx/neper/assets/162775882/c3f18a39-29de-4b58-90bc-305df156e008)
I hope to conduct research on the roughness of this surface.

**Comment by rquey** [comment, marked as answer]
Not that I can think of. I would work directly with the tessellation, ignore/remove all top grains and work on the properties of the surface formed by the bottom grains. You can do it in Neper itself or in an external program.

**EDIT**: you can also remove the top cells using `-transform rmcell` and work from there.
(needs version >= 4.9.1-2)
